yarn-project/simulator/src/avm/temporary_executor_migration.ts

```diff
@@ -0,0 +1,110 @@
+// All code in this file needs to die once the public executor is phased out.
+import { FunctionL2Logs } from '@aztec/circuit-types';
```

Comment:
Everything in this file will be phased out when the kernels change, getting into trunk will allow functionality to be gradually added